packages/flutter_markdown/lib/src/markdown_raw.dart

```diff
@@ -129,7 +129,25 @@ class _MarkdownBodyRawState extends State<MarkdownBodyRaw> {
 
   void initState() {
     super.initState();
+    _buildMarkdownCache();
+  }
+
+  void dispose() {
+    _linkHandler.dispose();
+    super.dispose();
+  }
+
+  void didUpdateConfig(MarkdownBodyRaw oldConfig) {
+    if (oldConfig.data != config.data
+      || oldConfig.markdownStyle != config.markdownStyle
+      || oldConfig.syntaxHighlighter != config.syntaxHighlighter
+      || oldConfig.onTapLink != config.onTapLink
```

Comment:
> This will cause spurious rebuilds. Instead, we should have the link handler call a member function on this object which calls through to the onTapLink function so that you don't need to rebuild when the closure changes.

It's not really that simple. See https://github.com/flutter/flutter/issues/2591